index.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var pino = require('pino'), // Can be overwritten for testing
```

Comment:
I'm curious as to why you chose `pino` over `winston`, which I've seen used in other clay/amphora-related projects?

Reply:
http://getpino.io/#/?id=benchmarks

Reply:
EXTREME

Reply:
It's a multipart answer. 

1. Pino is more performant in general
2. Winston accommodated transports as a main part of the primary process thread whereas Pino relies on piping to stdout and then using another process to transport. It's more performant and flexible.
3. Pino is configured with tools like ELK in mind and then pretty printing as a secondary priority